docs/dashboard/catalog/products.md

```diff
@@ -307,6 +327,14 @@ When you receive new stock, enter the amount in the Inventory field to keep up t
 
 You can repeat this process for all sizes or types of the same product by clicking the Variants card on the left side. Each variant has a specific SKU code (Stock Keep Unit) which is unique to specific variants.
 
+#### How to manage metadata in the variant creator page
```

Comment:
Variant Creator (the bulk one- where you create multiple variants at one after creating a new product) doesn't allow for setting metadata. For variants we can do that only on the variant details page.